mycroft/skills/diagnostics/vocab/en-us/IpKeyword.voc

```diff
@@ -0,0 +1,10 @@
+Ip
+What is my ip?
+What is my eye pea?
+What is my eye pee?
+What is my I pea?
+What is my I pee?
```

Comment:
I don't think that these and other .voc files should have question marks in them, as the stt seems to never return them. That may also be what is breaking the intent tests.